.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,4 @@
+- [ ] Closes #xxxx
+- [ ] Tests added / passed
+- [ ] Passes `black dask_ml` / `flake8 dask_ml`
```

Comment:
> We'll probably want to add isort and mypy to this check too. 

I was waiting for the resolution of https://github.com/dask/dask-ml/pull/813 first, before introducing an `isort` check here, but now that you've brought it up, I don't see why it couldn't be introduced at the current stage.
Thanks for reminding me about `mypy` - I'd completely forgotten about it.

> At a certain point it might be worth pointing to ci/code_checks.sh or pre-commit instead

Do you mean verbally, or is there some related GitHub magic trick(s)?